src/config.rs

```diff
@@ -0,0 +1,40 @@
+use crate::utils;
+use std::path::Path;
+use toml::Value;
+
+/**
+ * Extract the [build.target-dir] string from cargo config file
+ * 
+ * - Receive the workspace root path
+ */
+pub fn get_targer_dir_name(path: &Path) -> String {
+    // Check if the file exists
+    if let Some(config_file_path) = utils::find_cargo_config(&path).unwrap() {
+        // Get file content as string
+        let contents = std::fs::read_to_string(config_file_path).unwrap();
+        // Parse file content to toml::Value
+        let config = contents.parse::<Value>();
+
+        // Check if parsed sucessfully
+        if config.is_ok() {
+            let config = config.ok();
+
+            match config {
+                Some(c) => {
+                    // Check if [build] exists
+                    if let Some(build) = c.get("build") {
+                        // Check if [build] has target-dir
+                        if let Some(target_dir) = build.get("target-dir") {
+                            // Remove trailing "
+                            return target_dir.to_string().replace("\"", "");
```

Comment:
Any idea what this is about?

```toml
[build]
target-dir = "target"
```
That should result in the value `target`, not `target"` - then there's a bug in the toml parser that should be fixed instead!

Reply:
You might as well define a serde struct like `Manifest` instead of parsing to a `toml::Value`.

Reply:
When I create the target-dir string, it comes with the quotes, like this: "target". So I removed the quotes with:

 `.replace (" \ "") `, 

If not, a directory was created with the name "target", and the correct one would be just target.

About tailing ", it was a translation error, I meant: remove the two quotes, which were around the word.

I was unable to define a serde struct because the target-dir configuration is in kebab-case.

Reply:
Ah, it's pretty obvious what happens here: `to_string` gives a formatted representation of `toml::Value`. You have to call `.as_str()` to extrac the string, if that value was a string.

As mentioned just above, better to use a `#[derive(Deserialize)] struct`, that should make it easier to add more members in the future as well.

Reply:
> I was unable to define a serde struct because the target-dir configuration is in kebab-case.

Introducing `#[serde(rename = "target-dir")]` :)